core/domain/voiceover_services.py

```diff
@@ -509,3 +513,86 @@ def create_exploration_voice_artists_link_model_instance(
     exploration_voice_artists_link_model.update_timestamps()
 
     return exploration_voice_artists_link_model
+
+
+def update_exploration_voice_artist_link_model(
+    user_id: str,
+    change_list: Sequence[exp_domain.ExplorationChange],
+    old_exploration: exp_domain.Exploration,
+    updated_exploration: exp_domain.Exploration
+) -> None:
+    """Create or update a voice artist link model following modifications to
+    the recorded voiceover property in an exploration's state.
+
+    Args:
+        user_id: str. The committer ID for the given change list.
+        change_list: list(ExplorationChange). A list of exploration chnage
+            objects.
+        old_exploration: Exploration. An instance of exploration class
+            representing an old version.
+        updated_exploration: Exploration. An instance of exploration class
+            representing the latest version.
+    """
+
+    if not feature_flag_services.is_feature_flag_enabled(
```

Comment:
This feature flag needs to be truned on after the job has successfully ran in prod servers. 

Adding @seanlip for visibility. (I am updating the instruction doc for the beam job regarding this feature flag)